extensions/amp-mega-menu/amp-mega-menu.md

```diff
@@ -0,0 +1,228 @@
+---
```

Comment:
/fyi @nainar in case you would like to take a look.
/cc @CrystalOnScript for a documentation review